client/blocks/data-center-picker/index.tsx

```diff
@@ -23,297 +16,148 @@ interface ExternalProps {
 type Props = ExternalProps & LocalizeProps;
 
 const DataCenterOptions = [
+	{
+		value: '',
+		get label(): string {
+			return translate( 'Optimal data center' );
```

Comment:
Just for my learning: if this option is chosen, does the user see which optimal data center was chosen for them in the Server Settings -> Web Server Settings -> Primary Data Center field once the site is atomic?

Reply:
No, looks like we only show it if you've chosen a `geoAffinity` value: 

https://github.com/Automattic/wp-calypso/blob/7c0a42dda287bd6b8474d9d29eedb346a641e258/client/hosting/server-settings/components/web-server-settings-card/index.tsx#L227-L230

We could also try it out and make sure that's the expected behavior. 🙂

Reply:
Confirmed that the chosen data center is not visible in the UI.